api/tests/unit/infrastructure/serializers/jsonapi/certification-center-for-admin-serializer_test.js

```diff
@@ -2,110 +2,197 @@ import { expect, domainBuilder } from '../../../../test-helper.js';
+        });
 
-      // when
-      const serializedCertificationCenter = serializer.serialize(certificationCenter);
+        // when
+        const serializedCertificationCenter = serializer.serialize(certificationCenter);
 
-      // then
-      expect(serializedCertificationCenter).to.deep.equal({
-        data: {
-          type: 'certification-centers',
-          id: '12',
-          attributes: {
-            name: 'Centre des dés',
-            type: 'SCO',
-            'external-id': '12345',
-            'data-protection-officer-first-name': 'Justin',
-            'data-protection-officer-last-name': 'Ptipeu',
-            'data-protection-officer-email': 'justin.ptipeu@example.net',
-            'created-at': new Date('2018-01-01T05:43:10Z'),
-            'is-v3-pilot': false,
+        // then
+        expect(serializedCertificationCenter).to.deep.equal({
+          data: {
+            type: 'certification-centers',
+            id: '12',
+            attributes: {
+              ...certificationCenterJsonApi.data.attributes,
+              'created-at': new Date('2018-01-01T05:43:10Z'),
+              'is-v3-pilot': false,
+            },
+            relationships: {
+              'certification-center-memberships': {
+                links: {
+                  related: `/api/admin/certification-centers/${certificationCenter.id}/certification-center-memberships`,
+                },
+              },
+              habilitations: {
+                data: [
+                  {
+                    id: '1',
+                    type: 'complementary-certifications',
+                  },
+                ],
+              },
+            },
           },
-          relationships: {
-            'certification-center-memberships': {
-              links: {
-                related: `/api/admin/certification-centers/${certificationCenter.id}/certification-center-memberships`,
+          included: [
+            {
+              id: '1',
+              type: 'complementary-certifications',
+              attributes: {
+                key: 'SURF',
+                label: 'Pix+surf',
               },
             },
-            habilitations: {
-              data: [
-                {
-                  id: '1',
-                  type: 'complementary-certifications',
-                },
-              ],
+          ],
+        });
+      });
+    });
+  });
+
+  describe('when the center is for v3 certification', function () {
+    describe('#deserialize', function () {
+      it('should convert a JSON API data into a CertificationCenterForAdmin model object', function () {
+        // when
+        const deserializedCertificationCenterForAdmin = serializer.deserialize({
+          data: {
+            type: 'certification-centers',
+            id: '123',
+            attributes: {
+              ...certificationCenterJsonApi.data.attributes,
+              'is-v3-pilot': true,
             },
+            relationships: {},
```

Comment:
```suggestion
       certificationCenterJsonApi.data.attributes['is-v3-pilot'] = true;
        const deserializedCertificationCenterForAdmin = serializer.deserialize(certificationCenterJsonApi)
```

Et je pense qu'on peut appliquer cette logique à d'autres endroits :)